src/ucp/proto/proto_am.inl

```diff
@@ -63,7 +62,6 @@ ucs_status_t ucp_do_am_bcopy_multi(uct_pending_req_t *self, uint8_t am_id_first,
             return packed_len; /* Failed */
         }
 
-        req->send.state.offset += packed_len - hdr_size_middle;
```

Comment:
@yosefe The `offset` handling moved into `ucp_tag_pack_eager_dt_copy()`. Please review.